client/app/bundles/course/user-invitations/operations.ts

```diff
@@ -129,3 +130,43 @@ export function toggleRegistrationCode(shouldEnable: boolean): Operation {
         );
       });
 }
+
+export const splitEntries = (input: string): string[] => {
+  return input.split(/\s*[;,]\s*(?=(?:[^"]*"[^"]*")*[^"]*$)/);
```

Comment:
consider updating to allow split by newlines too `/\s*[;,\n]\s*(?=(?:[^"]*"[^"]*")*[^"]*$)/`

Reply:
done